packages/cli/generators/model/index.js

```diff
@@ -204,18 +204,18 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
     // Set up types for Templating
     const TS_TYPES = ['string', 'number', 'object', 'boolean', 'any'];
     const NON_TS_TYPES = ['geopoint', 'date'];
-    Object.entries(this.artifactInfo.properties).forEach(([key, val]) => {
+    Object.values(this.artifactInfo.properties).forEach(val => {
       // Default tsType is the type property
       val.tsType = val.type;
 
       // Override tsType based on certain type values
       if (val.type === 'array') {
-        if (TS_TYPES.includes(val.arrayType)) {
-          val.tsType = `${val.arrayType}[]`;
+        if (TS_TYPES.includes(val.itemType)) {
+          val.tsType = `${val.itemType}[]`;
         } else if (val.type === 'buffer') {
-          val.tsType = `Buffer[]`;
+          val.tsType = 'Buffer[]';
         } else {
-          val.tsType = `string[]`;
+          val.tsType = 'string[]';
```

Comment:
'string[]' ---> 'String[]' ?

Reply:
I know `string[]` works but does `String[]` work as well?

Reply:
sorry never mind :( I mixed it with the array type in juggler. `string[]` is correct.

Reply:
found a `String[]` on line https://github.com/strongloop/loopback-next/pull/1570/files#diff-328c8e879ea3a53d7fb50599f1c6a77eR98...I feel confused again 🤔

Reply:
I _think_ they both work 🤷‍♂️